### PR TITLE
Refresh homepage

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

**BLOCKED** by [this brand issue](https://warthogs.atlassian.net/browse/WD-1195)
- Refreshes homepage based on changes in the [copydoc](https://docs.google.com/document/d/1-QO5zAclj469qGvFmNJBjQ_qqI_T_uliwj6mAo8ZbCs/edit#)
- An issue has been created to address Lyubo's comments on inconsistant stroke thickness that can be tracked [here](https://warthogs.atlassian.net/browse/WD-1195)

## QA

- Go to https://microstack-run-214.demos.haus/ and compare against [copydoc](https://docs.google.com/document/d/1-QO5zAclj469qGvFmNJBjQ_qqI_T_uliwj6mAo8ZbCs/edit#)
- Check that all six individual tabs work and don't interfere with each other

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/6311

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -25,173 +25,172 @@ <h1> OpenStack for the edge, <br/> micro clouds and developers </h1>
...
       </div>
   </section>
+
   <section class="p-strip">
     <div class="row">
       <div class="p-notification--caution">
         <div class="p-notification__content">
           <p class="p-notification__message">
-            <strong>Note:</strong> MicroStack is in a beta state. We encourage you to test it, give us your <a href="https://bugs.launchpad.net/microstack">feedback</a> and <a href="https://discourse.juju.is/search?q=microstack">ask questions</a>. For information about commercial support for MicroStack, <a href="https://ubuntu.com/openstack/contact-us">contact Canonical</a>.
+            <strong>Note:</strong> MicroStack is in a beta state. We encourage you to test it, share your <a href="https://microstack.run/survey">feedback</a> and <a href="https://discourse.juju.is/search?q=microstack">ask questions</a>. For commercial support, <a href="/openstack/contact-us">contact us directly</a>.
```
Comment (anthonydillon) [inline templates/index.html:34]: Could you use the [Information Notification](https://vanillaframework.io/docs/patterns/notification#information) pattern here please?

File: templates/partial/_navigation.html [hunk 2]
```diff
@@ -21,13 +21,13 @@
...
           <li class="p-navigation__item" role="menuitem">
-            <a class="p-navigation__link" href="https://opendev.org/x/microstack">Contribute</a>
+            <a class="p-navigation__link" href="https://github.com/openstack-snaps/snap-microstack">Contribute</a>
           </li>
           <li class="p-navigation__item" role="menuitem">
             <a class="p-navigation__link" href="https://bugs.launchpad.net/microstack/+filebug">Report a bug</a>
           </li>
           <li class="p-navigation__item" role="menuitem">
-            <a class="p-navigation__link" href="https://discourse.charmhub.io/search?q=microstack">Community</a>
+            <a class="p-navigation__link" href="https://discourse.charmhub.io/search?q=microstack">Get community help</a>
```
Comment (anthonydillon) [inline templates/partial/_navigation.html:30]:
```suggestion
            <a class="p-navigation__link" href="https://discourse.charmhub.io/tag/microstack">Get community help</a>
```

File: templates/index.html (hunk 1 above)
Comment (anthonydillon) [inline templates/index.html:34]:
```suggestion
            <strong>Note:</strong> MicroStack is in a beta state. We encourage you to test it, share your <a href="https://microstack.run/survey">feedback</a> and <a href="https://discourse.charmhub.io/tag/microstack">ask questions</a>. For commercial support, <a href="https://ubuntu.com/openstack/contact-us">contact us directly</a>.
```

File: templates/index.html [hunk 3]
```diff
@@ -25,173 +25,172 @@ <h1> OpenStack for the edge, <br/> micro clouds and developers </h1>
...
+      <p>Install OpenStack anywhere in a few simple steps and let <a href="/engage/kubernetes-operators-explained-whitepaper">Kubernetes operators</a> manage it for you. MicroStack is NOT yet another OpenStack on Kubernetes. It is a canonical OpenStack with native Kubernetes experience.</p>
+    </div>
     <div class="row">
-      <div class="col-8" style="margin-top: 2rem;">
-        <h2>Straightforward installation</h2>
-        <h3 class="p-heading--4">Fully functional OpenStack in 2 commands and ~20 minutes.</h3>
+      <div class="col-6 p-card">
+        <h3>Lucid interface</h3>
+        <hr class="u-sv1">
+        <p>MicroStack is the most straightforward OpenStack ever. It installs in minutes through a friendly interface, making OpenStack fully accessible to those with no previous experience. Just <a href="/openstack/tutorials">give it a try</a> and see whether OpenStack is suitable for you.</p>
```
Comment (anthonydillon) [inline templates/index.html:47]: There is no link in the copy doc.

File: templates/index.html [hunk 4]
```diff
@@ -25,173 +25,172 @@ <h1> OpenStack for the edge, <br/> micro clouds and developers </h1>
...
-      <div class="col-4 p-divider__block u-vertically-center">
-        <h3 class="u-align--center">7 contributors</h3>
-      </div>
-      <div class="col-4 p-divider__block u-vertically-center">
-        <h3 class="u-align--center">~250 commits</h3>
-      </div>
-      <div class="col-4 p-divider__block u-vertically-center">
+    <div class="u-fixed-width">
+      <div class="p-divider__block u-vertically-center">
         <h3 class="u-align--center" style="margin-left: 1.5rem; margin-right: 1.5rem;">~1,500 active installations</h3>
```
Comment (anthonydillon) [inline templates/index.html:70]: You don't need the inline styles here

Comment (anthonydillon) [reply templates/index.html:70]: Is this a header of a p with a heading class?

File: templates/index.html [hunk 5]
```diff
@@ -25,173 +25,172 @@ <h1> OpenStack for the edge, <br/> micro clouds and developers </h1>
...
-      </div>
-  </section>
-  <section class="p-strip--suru is-dark">
     <div class="row">
       <div class="col-6">
-        <h2>Share your use case</h2>
-        <p>Web applications, AI/ML or maybe telco NFV? What are you using MicroStack for? Do not hesitate and share your use case, and benefit from the community collaboration and help.</p>
+        <h2>OpenStack for developers</h2>
+        <h3 class="p-heading--4">On your workstation; in your CI/CD environment</h3>
+        <p>Designed for devices with minimal hardware resources, MicroStack is perfectly suitable for developer workstations and <a href="/engage/devops-cicd-webinar">CI/CD</a> environments.</p>
```
Comment (anthonydillon) [inline templates/index.html:80]:
```suggestion
        <p>Designed for devices with minimal hardware resources, MicroStack is perfectly suitable for developer workstations and <a href="https://ubuntu.com/engage/devops-cicd-webinar">CI/CD</a> environments.</p>
```

File: templates/index.html [hunk 6]
```diff
@@ -25,173 +25,172 @@ <h1> OpenStack for the edge, <br/> micro clouds and developers </h1>
...
+      </div>
       <div class="col-6">
-        <h2 class="u-sv2">Open source cloud</h2>
-        <p>MicroStack is a pure upstream OpenStack distribution which effectively decreases virtualisation costs. It is a reasonable alternative to VMware clusters, Microsoft Hyper-V, Citrix Hypervisor and Proxmox Virtualization Environment.</p>
+        <h2>OpenStack for micro clouds</h2>
+        <h3 class="p-heading--4">Small-scale cost-effective private cloud infrastructure</h3>
+        <p>OpenStack private cloud implementation used to be challenging due to its complexity and scale. This is no longer the case.</p>
+        <p>MicroStack enables enterprises to quickly deploy fully functional private cloud infrastructure that serves as a cost-effective alternative to VMware vSphere, Citrix Hypervisor and Proxmox Virtual Environment.</p>
+        <p>
+          <a href="/openstack/features">For large-scale deployments, try Charmed OpenStack&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/index.html:117]:
```suggestion
          <a href="https://ubuntu.com/openstack/features">For large-scale deployments, try Charmed OpenStack&nbsp;&rsaquo;</a>
```

File: templates/index.html [hunk 7]
```diff
@@ -25,173 +25,172 @@ <h1> OpenStack for the edge, <br/> micro clouds and developers </h1>
...
-        <hr class="u-sv1">
-        <p>Single-node installation is usually the first step to try OpenStack. MicroStack supports this use case, providing the ability to deploy fully functional OpenStack on a single machine with just 2 commands.</p>
-        <p><a href="/docs/single-node">Read single-node installation instructions&nbsp;&rsaquo;</a></p>
+      <div class="col-6">
+        <h2>OpenStack for cloud providers</h2>
+        <h3 class="p-heading--4">Open-source equivalent to AWS Outposts and Azure Stack</h3>
+        <p>Cloud providers who operate an OpenStack public cloud usually need an extension of their infrastructure running on customer premises for privacy and data sovereignty reasons.</p>
+        <p>MicroStack answers these needs, delivering an out-of-the-box cloud solution with native OpenStack APIs that can be easily installed or shipped pre-installed to end users.</p>
+        <p>
+          <a href="/engage/public-cloud-openstack-implementation">Watch a webinar about public cloud implementation&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/index.html:129]:
```suggestion
          <a href="https://ubuntu.com/engage/public-cloud-openstack-implementation">Watch a webinar about public cloud implementation&nbsp;&rsaquo;</a>
```

File: templates/index.html [hunk 8]
```diff
@@ -25,173 +25,172 @@ <h1> OpenStack for the edge, <br/> micro clouds and developers </h1>
...
+            ) | safe
+          }}
+      </div>      
+      <div class="col-6">
+        <h2>OpenStack for the edge</h2>
+        <h3 class="p-heading--4">Comprehensive cloud platform for distributed low-latency applications</h3>
+        <p>MicroStack answers the needs of multi-access edge computing (MEC), providing a secure, reliable and scalable cloud platform with a minimal footprint and simplified lifecycle management capabilities.</p>
+        <p>Ideal to meet requirements in telecom, industrial, automotive and other market sectors that need <a href="/edge-computing">edge infrastructure</a>.</p>
+        <p>
+          <a href="/engage/edge-infrastructure">Read a whitepaper about OpenStack on the edge&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/index.html:163]:
```suggestion
          <a href="https://ubuntu.com/engage/edge-infrastructure">Read a whitepaper about OpenStack on the edge&nbsp;&rsaquo;</a>
```

File: templates/index.html [hunk 9]
```diff
@@ -25,173 +25,172 @@ <h1> OpenStack for the edge, <br/> micro clouds and developers </h1>
...
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+      </div>      
+      <div class="col-6">
+        <h2>OpenStack for the edge</h2>
+        <h3 class="p-heading--4">Comprehensive cloud platform for distributed low-latency applications</h3>
+        <p>MicroStack answers the needs of multi-access edge computing (MEC), providing a secure, reliable and scalable cloud platform with a minimal footprint and simplified lifecycle management capabilities.</p>
+        <p>Ideal to meet requirements in telecom, industrial, automotive and other market sectors that need <a href="/edge-computing">edge infrastructure</a>.</p>
```
Comment (anthonydillon) [inline templates/index.html:161]:
```suggestion
        <p>Ideal to meet requirements in telecom, industrial, automotive and other market sectors that need <a href="https://ubuntu.com/edge-computing">edge infrastructure</a>.</p>
```

File: templates/index.html [hunk 10]
```diff
@@ -208,10 +207,54 @@ <h3>What OS are you on?</h3>
...
+        <p>(less steps; less capabilities)</p>          
+      </a>
+
+      <div class="col-12 js-tab__content" id="tab-four__content" role="tabpanel" aria-labelledby="tab-four">           
         <ol class="p-stepped-list u-no-margin--left">
+          <li class="p-stepped-list__item">
+            <h3 class="p-stepped-list__title u-sv2">Install MicroStack</h3>
+            <div class="p-stepped-list__content">
+              <p>
+                <a href="https://discourse.ubuntu.com/t/install-and-configure-microstack-sunbeam-single-node/32157" class="p-button--positive">Follow a tutorial</a>
```
Comment (anthonydillon) [inline templates/index.html:246]: Is this all? Seems there is more content but incomplete in the copy doc

Comment (anthonydillon) [reply templates/index.html:246]: Might want to check with @tytus-kurek if the instructions section is correct.

File: templates/index.html [hunk 11]
```diff
@@ -358,34 +527,47 @@ <h3 class="p-stepped-list__title u-sv2">Launch a VM</h3>
...
   <section class="p-strip">
     <div class="row u-vertically-center">
-      <div class="col-6">
-        <h2>The team behind MicroStack</h2>
-        <p>MicroStack is maintained by the <a href="https://ubuntu.com/openstack">OpenStack team at Canonical</a>. We provide OpenStack packages on Ubuntu and have two OpenStack distributions of our own. MicroStack is an on-rails, opinionated OpenStack for the edge, micro clouds and developers. <a href="https://ubuntu.com/openstack/features">Charmed OpenStack</a> is a composable enterprise OpenStack for large scale deployments in data centres.</p>
+      <div class="col-7">
+        <h2>Need more composable OpenStack?</h2>
+        <p><a href="/openstack/features">Try Charmed OpenStack</a>, a distribution that gives you total control of all OpenStack components and their permutations. Build your cloud from the ground up and tailor it to your needs.</p>
+        <p>
+          <a href="/openstack/install">Get started with Charmed OpenStack&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/index.html:546]:
```suggestion
          <a href="https://openstack/install">Get started with Charmed OpenStack&nbsp;&rsaquo;</a>
```

File: templates/index.html [hunk 12]
```diff
@@ -358,34 +527,47 @@ <h3 class="p-stepped-list__title u-sv2">Launch a VM</h3>
...
   </section>
+
   <section class="p-strip">
     <div class="row u-vertically-center">
-      <div class="col-6">
-        <h2>The team behind MicroStack</h2>
-        <p>MicroStack is maintained by the <a href="https://ubuntu.com/openstack">OpenStack team at Canonical</a>. We provide OpenStack packages on Ubuntu and have two OpenStack distributions of our own. MicroStack is an on-rails, opinionated OpenStack for the edge, micro clouds and developers. <a href="https://ubuntu.com/openstack/features">Charmed OpenStack</a> is a composable enterprise OpenStack for large scale deployments in data centres.</p>
+      <div class="col-7">
+        <h2>Need more composable OpenStack?</h2>
+        <p><a href="/openstack/features">Try Charmed OpenStack</a>, a distribution that gives you total control of all OpenStack components and their permutations. Build your cloud from the ground up and tailor it to your needs.</p>
```
Comment (anthonydillon) [inline templates/index.html:544]:
```suggestion
        <p><a href="https://ubuntu.com/openstack/features">Try Charmed OpenStack</a>, a distribution that gives you total control of all OpenStack components and their permutations. Build your cloud from the ground up and tailor it to your needs.</p>
```

File: templates/index.html [hunk 13]
```diff
@@ -341,10 +509,11 @@ <h3 class="p-stepped-list__title u-sv2">Launch a VM</h3>
         </ol>
         <p>
           That's it! Your micro cloud is ready. You can now follow with <a href="https://ubuntu.com/openstack/tutorials" >MicroStack tutorials</a>.
-        </p>
-      </div>
+        </p>        
+      </div>
```
Comment (anthonydillon) [inline templates/index.html:513]: You editor should strip whitepace on save

File: templates/index.html [hunk 14]
```diff
@@ -1,8 +1,8 @@
 {% extends 'base_layout.html' %}
-{% block title %}OpenStack for the edge, micro clouds and developers{% endblock %}
+{% block title %}OpenStack on Kubernetes | Ubuntu{% endblock %}
```
Comment (anthonydillon) [inline templates/index.html:2]: This title is not updated, might be an issue with the base template

File: templates/index.html [hunk 15]
```diff
@@ -1,8 +1,8 @@
...
 
 {% block description %}
-MicroStack is a pure upstream OpenStack platform, designed for the edge and small-scale private cloud deployments, that can be installed and maintained with a minimal effort.
+MicroStack delivers distilled OpenStack excellence with native K8s experience. OCI images, Kubernetes operators, lucid interface - all you need to run fully functional OpenStack on Kubernetes with minimal effort.
```
Comment (anthonydillon) [inline templates/index.html:5]: Same here, check the base template is not hardcoded.

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Good work @petesfrench. A few comments in the code.

**Comment by lyubomir-popov** [comment]
image
image
image
are the above new additions, or already there? The stroke thickness varies a lot. If they were already there, then maybe we ca nfix this once planning unit provide an updated icon style. If we're adding any of these in this pr, I would suggest sending them to our current icon designer to conform stroke thicknesses.

File: templates/partial/_navigation.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (petesfrench) [reply templates/partial/_navigation.html:30]: I have updated the link to the one in the copydoc, is that all that was needed? I can't see any comments

**Comment by juanruitina** [comment]
Two tweaks for better accessibility:

- Can you please add `Meet MicroStack - YouTube` as the `title` attribute for the video iframe?
- I see subtitles below some headings (e.g. "On your workstation; in your CI/CD environment") are `h3` elements, but technically they are not headings. Can you make them `p` elements instead? Styling still applies via the class, so it should be fine.

I noticed `MicroStack` is not properly capitalised in one instance, under "K8s-native architecture". I edited the copy doc to fix this.

Number of active installations has increased to "~2,300 active installations". Besides, I think that banner looks rather weak now that it is the single figure given as social proof. I wonder if something could be done at the visual level (@lyubomir-popov), or perhaps place the copy somewhere else and get rid of the banner altogether. E.g. we could incorporate it to the hero text, e.g. "Distilled upstream excellence with a native K8s experience. Already used in over 2,300 installations." What do you think?

**Comment by petesfrench** [comment]
@lyubomir-popov These were mostly additions in this PR. Who is the person ask about this?

**Comment by petesfrench** [comment]
Thanks @juanruitina, I have applied those changes. Tytus was also happy with removing the banner. Could you take another look and give it a +1 if you are happy?

File: templates/partial/_navigation.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (anthonydillon) [reply templates/partial/_navigation.html]: That is it.

**Comment by juanruitina** [comment]
LGTM now!

**Comment by anthonydillon** [comment]
Single QA question:
- [ ] Missing the "~2,300 active installations" section

**Review by anthonydillon — APPROVED** [review]
The code looks good with QA questions.

**Comment by petesfrench** [comment]
> Single QA question:
> 
> * [x]  Missing the "~2,300 active installations" section

@anthonydillon It was agreed with Juan and Tytus to remove the banner, at least for now. I will add that information to the description.

**Comment by petesfrench** [comment]
@lyubomir-popov I have updated the SVGs, can you take another look when you have time please?

**Comment by lyubomir-popov** [comment]
Hi @petesfrench much better, two questions:
- are the images displayed at the actual size of the files? Most seem to be 250px tall, with some exceptions. If  those are the sizes then great.
- This one seems blurred - might be worth checking again 
image

**Pull request merged**